blackbox/docs/general/dml.rst

```diff
@@ -491,12 +491,12 @@ about failures and successfully imported records is returned.
 ::
 
    cr> COPY locations FROM '/tmp/import_data/locations_with_failure/locations*.json' RETURN SUMMARY;
-    +--...--+----------...--------+---------------+-------------+------------------------------------------------------------------+
-    | node  | uri                 | success_count | error_count | errors                                                           |
-    +--...--+----------...--------+---------------+-------------+------------------------------------------------------------------+
-    | {...} | .../locations1.json |             6 |           0 | {}                                                               |
-    | {...} | .../locations2.json |             5 |           2 | {"failed to parse [date]": {"count": 2, "line_numbers": [1, 2]}} |
-    +--...--+----------...--------+---------------+-------------+------------------------------------------------------------------+
+    +--...--+----------...--------+---------------+-------------+-------------------...--------------------------------------+
+    | node  | uri                 | success_count | error_count | errors                                                     |
+    +--...--+----------...--------+---------------+-------------+-------------------...--------------------------------------+
+    | {...} | .../locations1.json |             6 |           0 | {}                                                         |
+    | {...} | .../locations2.json |             5 |           2 | {"failed to parse ...{"count": 2, "line_numbers": [1, 2]}} |
+    +--...--+----------...--------+---------------+-------------+-------------------...--------------------------------------+
```

Comment:
isn't this unrelated to the ES upgrade?

Reply:
No, the error message has become more detailed and includes additional information.